plugins/test/__snapshots__/post_type.test.jsx.snap

```diff
@@ -0,0 +1,104 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
This snapshot is not updated because I'm not able to execute the tests properly yet.